duckduckgo/iOS
# Do not sell implementation

Task/Issue URL: https://app.asana.com/0/0/1191298152080292/f
Tech Design URL: https://app.asana.com/0/481882893211075/1195505979185439/f
CC: @brindy @catehstn @bwaresiak 

**Description**:
This implements the Do Not Sell (DN$) signal in the browser. There is a new setting labeled "Global Privacy Control" in the settings page. Turning this on will send the `Sec-GPC` header on main document requests to '1' and set the `navigator.globalPrivacyControl` property to `true`.

**Steps to test this PR**:
1. Navigate to http://global-privacy-control.glitch.me
2. Initially the header and DOM sections should say not present
3. Turn on the "Global Privacy Control" setting
4. The webpage will refresh and the header and DOM sections of the web page should say present

1. Navigate to another site like twitter.com
2. Manually inspect the webview via safari and check the header is included in the main document request and log the value of `navigator.globalPrivacyControl` to ensure it is `true`



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
